js/views/userPageVw.js

```diff
@@ -256,7 +256,35 @@ module.exports = baseVw.extend({
       if (e.guid === this.model.get('page').profile.guid) {
         this.renderUserUnblocked();
       }
-    });        
+    });  
+    
+    this.listenTo(window.obEventBus, 'saveCurrentForm', function(response){      
+      switch (response) {
+        case 'item':
+          self.saveItem();
+          break;
+        case 'customize':
+          self.saveCustomizePage();
+          break;
+      }
+    });   
+    
+    $(window).off('keydown.view').bind('keydown.view', function(e) {
```

Comment:
same as above, this handler needs to be removed in the view's remove() function.